TOOLS/osxbundle.py

```diff
@@ -80,7 +80,7 @@ def main():
 
     if options.deps:
         print("> bundling dependencies")
-        sh(" ".join(["TOOLS/dylib-unhell.py", target_binary(binary_name)]))
+        print(sh(" ".join(["TOOLS/dylib-unhell.py", target_binary(binary_name)])))
```

Comment:
Not related to this change that you've done here, but we might want to get rid of this space-joining at some point, since I think one of the Popen-related constructors/functions certainly took in arrays of strings for arguments instead of a single string :) .